lib/src/builder.dart

```diff
@@ -177,9 +212,12 @@ class PostgrestQueryBuilder extends PostgrestBuilder {
   /// ```dart
   /// postgrest.rpc('get_status', { name_param: 'supabot' })
   /// ```
-  PostgrestBuilder rpc(dynamic params) {
+  PostgrestBuilder rpc(dynamic params, {bool head = false}) {
```

Comment:
we don't need this anymore

Reply:
Can't believe I missed that. Thanks for pointing out!